jumpscale/packages/admin/frontend/api.js

```diff
@@ -193,6 +193,16 @@ const apiClient = {
             return axios({
                 url: `${baseURL}/admin/clear_blocked_nodes`,
             })
+        },
+        getNotifications: () => {
+            return axios({
+                url: `${baseURL}/admin/get_notifications`,
+            })
+        },
+        getNotificationsCount: () => {
```

Comment:
what about calculating the count from the `getNotifications` instead of doing another request to just get the count

Reply:
We just use this request to check whether we should request the notifications or not. Instead of always requesting old notifications